egps/egp-5/egp-5.ts

```diff
@@ -43,25 +45,25 @@ async function proposal() {
    */
 
   // Deploy the airdrop contract
-  const airdropDeployer = new ethers.ContractFactory(
-    airdropData.abi,
-    airdropData.bytecode,
-    signer
-  )
-  const airdropContract = await airdropDeployer.deploy(
-    addresses.CoreVoting,
-    hexRoot,
-    '0x5c6D51ecBA4D8E4F20373e3ce96a62342B125D6d', // ELFI Contract address
-    expiration.getTime(),
-    '0x02Bd4A3b1b95b01F2Aa61655415A5d3EAAcaafdD' // locking vault address
-  )
+  // const airdropDeployer = new ethers.ContractFactory(
+  //   airdropData.abi,
+  //   airdropData.bytecode,
+  //   signer
+  // )
+  // const airdropContract = await airdropDeployer.deploy(
+  //   addresses.CoreVoting,
+  //   hexRoot,
+  //   '0x5c6D51ecBA4D8E4F20373e3ce96a62342B125D6d', // ELFI Contract address
+  //   expiration.getTime(),
+  //   '0x02Bd4A3b1b95b01F2Aa61655415A5d3EAAcaafdD' // locking vault address
+  // )
```

Comment:
Instead of commenting this out, can you make a second file `egp-5-patch.ts` or similar. that way we have one that shows the correct code and the other with this commented out?